cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -4509,16 +4509,16 @@ display.page.search.tab = statistics
 display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
-# This search captures certificates that will expire within 7 days, and if they are expiring within one day, the severity will be critical, otherwise, the severity will be high.
+# This search captures certificates that will expire within 30 days, and if they are expiring within seven day, the severity will be critical, otherwise, the severity will be high.
 search = `cs_windows_cert_store_idx` sourcetype="windows:certstore:local" \
 | dedup host SerialNumber \
 | `cs_windows_cert_time_format(NotAfter)` \
 | `cs_windows_cert_time_format(NotBefore)` \
 | eval interval_start=relative_time(now(),"-1m") \
-| eval interval_one_day=relative_time(now(),"+1d") \
-| eval interval_end=relative_time(now(),"+7d") \
+| eval interval_end=relative_time(now(),"+30d") \
+| eval interval_seven_day=relative_time(now(),"+7d") \
 | where interval_start<NotAfter AND NotAfter<interval_end \
```

Comment:
We need to test on following conditions with makeresults command.

* Only Once when certificate is about to expire in 30 days - high
* Only Once when certificate is about to expire in 7 days - critical
* Only Once when certificate is about to expire in 1 day - critical